content/Projects/Python/prom-api-client-python.md

```diff
@@ -0,0 +1,15 @@
+# prometheus-api-client/python
```

Comment:
Can we move it directly to `python-prometheus-api-client`? (: No  need for grouping

Reply:
updated :+1: